client/src/App.js

```diff
@@ -1,11 +1,26 @@
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Home from "./components/Home";
+import About from "./components/About";
+import Profile from "./components/Profile";
 function App() {
   return (
+    <>
     <div className="App">
       <header className="App-header">
         <h1>Welcome to Gymkhana IITI</h1>
       </header>
-    </div>
+     </div>
+
+    <Router>
+      <Routes>
+        <Route path="/home" element={<Home />}/>
```

Comment:
this sytax is wrong 
it should be element={Home}
similarly for other routes

Reply:
done as directed